tests/unit/teststatusicon.cpp

```diff
@@ -10,64 +10,49 @@
 void TestStatusIcon::basic() {
   StatusIcon si;
 #if defined(MVPN_LINUX) || defined(MVPN_WINDOWS)
-  QCOMPARE(si.iconUrl().toString(), "qrc:/ui/resources/logo-generic.png");
```

Comment:
You can still keep these tests

Reply:
The method `iconUrl` is not used anymore.